Compiler/LangServer/Text/Buffer.cs

```diff
@@ -0,0 +1,35 @@
+using System.Collections.Generic;
+using System.Linq;
+using Core.Lexer.Tokenization.Models;
+using Core.Meta;
+using OmniSharp.Extensions.LanguageServer.Protocol.Models;
+
+namespace Compiler.LangServer
+{
+    public sealed class Buffer
+    {
+        public BebopSchema? Schema { get; }
+        public int? Version { get; }
+        public string Text { get; }
+        public List<TextLine> Lines { get; }
+        public int Length { get; }
+
+        public Buffer(BebopSchema? schema, string text, int? version)
+        {
+            Schema = schema;
+            Text = text;
+            Version = version;
+            Lines = TextCursor.Split(text);
+            Length = Lines.Sum(x => x.Length + x.LineBreak.Length);
+        }
+
+        public Range GetRange(Span span)
+        {
+            return new Range
+            {
+                Start = new Position(span.StartLine, span.StartColumn),
+                End = new Position(span.EndLine, span.EndColumn),
+            };
+        }
```

Comment:
Is this redundant to the definition in [Compiler/LangServer/Extensions/SpanExtensions.cs](https://github.com/RainwayApp/bebop/pull/197/files#diff-f0e26fca888de9503511986f9e50d8a6268a475b928c95630b1cbb203141c27c)?

Reply:
Yes, will fix this 👍